src/picologging/logger.cxx

```diff
@@ -27,6 +27,46 @@ int getEffectiveLevel(Logger*self){
     return LOG_LEVEL_NOTSET;
 }
 
+void setEnabledLevels(Logger *self) {
+    switch (getEffectiveLevel(self)){
+        case LOG_LEVEL_DEBUG:
```

Comment:
I think this can be refactored to setting all to false once and then using a fall-through switch statement like before.
That should reduce the size of this function (and the copy+pasting)

Reply:
I thought about this before but it won't reduce the size much as only higher levels will be smaller, but also reduces readability.
I've updated it as requested.

Reply:
You're right, there were some unnecessary changes for that. We only need to set all to false once and let the fall-through work.